src/views/databases/upload-database-modal.tsx

```diff
@@ -96,6 +98,7 @@ export const UploadDatabaseModal: FC<UploadDatabaseModalProps> = ({ onClose, onC
                         <SingleSelectOption key={group.name} label={group.name} value={group.name} />
                     ))}
                 </SingleSelectField>
+                <InputField className={styles.field} label="Prefix" value={prefix} onChange={({ value }) => setPrefix(value)} disabled={loading} />
```

Comment:
Wouldn't path be more user friendly than prefix?

Reply:
Good point, I was wondering about the same. "Path" , "Path prefix", "Prefix" ... I went with the latter as this is how I named the field in the API, but probaby just "Path" or "Path prefix" would be better. 🤷

Reply:
I would stick with path, that's what it is